### DISCUSSION TITLE
是否支持PHP空间搭建？

### DISCUSSION THREAD

**Discussion by pynne** [SeaHOH/GotoX#148] [点子]
是否支持PHP空间搭建？ 因为这样很便宜。现在php一般支持php7以上

**Comment by SeaHOH** [comment]
GotoX 不以收费服务为目标，如果你发现有其它稳定的免费服务，欢迎提供相应信息，我十分乐意添加支持。

**Comment by Quantum2Bits** [comment]
Heroku 的PHP是免费的。原来有些支持Heroku的软件已经不更新了。

**Comment by SeaHOH** [comment]
免费 Heroku 应用启动太慢。

**Reply by ZK6100HGA** [reply]
kubesail怎么样？

**Reply by SeaHOH** [reply]
流量太少，kubesail 不适合作为代理使用。

**Reply by ZK6100HGA** [reply]
> 
> 
> 流量太少，kubesail 不适合作为代理使用。

vercel适合吗？

**Reply by SeaHOH** [reply]
> vercel适合吗？

不允许代理
> ### Never Fair Use
>
>    :x: Proxies and VPNs
    :x: Media hosting for hot-linking
    :x: Scrapers
    :x: Crypto Mining
    :x: CPU-intensive APIs (e.g.: Machine Learning)
    :x: Load Testing
